src/analysis/Environments/Environments.test.ts

```diff
@@ -106,7 +104,7 @@ const getEnvironmentsProps = (propsOverrides?: Partial<EnvironmentsProps>): Envi
   return finalizedProps;
 };
 
-describe('Environments', () => {
+describe('Environments Component', () => {
```

Comment:
Nit: most tests label the describe block with the component name.

```suggestion
describe('Environments', () => {
```

Reply:
definitely nit, but I prefer the added clarity of specifying it's a component we are testing... Names like DeleteAppModal are more obviously a component, but 'Environments' benefits here from a bit more 'describe' 😃

Reply:
My vote would be to always follow the convention, but I won't request changes over it.